internal/store/bigtable/reader.go

```diff
@@ -109,12 +111,11 @@ func ReadWithFilter(
 // needed by the pagination APIs.
 func ReadWithGroupRowList(
 	ctx context.Context,
-	btGroup *Group,
+	tables []*cbt.Table,
 	prefix string,
 	accs []*Accessor,
 	unmarshalFunc func([]byte) (interface{}, error),
 ) ([][]BtRow, error) {
-	tables := btGroup.Tables(nil)
```

Comment:
note here use "nil" filter, but it should use the same filter as caller. 

But better to pass in the filtered tables directly here.